app/views/queue_items/index.html.haml

```diff
@@ -0,0 +1,38 @@
+%section.my_queue.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      %article
+        %header
+          %h2 My Queue
+        %table.table
+          %thead
+            %tr
+              %th(width="10%") List Order
+              %th(width="30%") Video Title
+              %th(width="10%") Play  
+              %th(width="20%") Rating 
+              %th(width="15%") Genre 
+              %th(width="15%") Remove 
+          %tbody 
+            - @queue_items.each do |queue_item|
+              %tr 
+                %td 
+                  = text_field :position, "List Order", value: queue_item.position 
+                %td
+                  = link_to queue_item.video_title, queue_item.video 
+                %td 
+                  = button_to "Play", nil, class: "btn btn-default"
+                %td 
+                  = select :queue_item, :rating, (1..5).map{|num| ["#{num} Stars", num] }, { include_blank: false }, class:'form-control'
```

Comment:
You should use `#pluralize` method for Rating and stars. Check out Rails API. Also it would be a good point to move that to a helper method.